build.gradle

```diff
@@ -7,52 +7,76 @@
 // PLUGINS
 // -----------------------------------------------------------------------------
 
+buildscript {
+    repositories {
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
+    }
+    dependencies {
+        classpath 'io.github.gradle-nexus:publish-plugin:1.3.0'
+    }
+}
 plugins {
     // Apply the brightSPARK Labs plugin for standardisation
-    id 'com.brightsparklabs.gradle.baseline' version "0.1.0"
+    id 'com.brightsparklabs.gradle.baseline' version "4.0.0"
 
     // Apply the java-library plugin to add support for Java Library
     id 'java-library'
 
     // Apply the groovy plugin to also add support for Groovy (needed for Spock)
     id 'groovy'
 
-    // Apply bintray plugin for publishing artifacts
-    id "com.jfrog.bintray" version "1.8.4"
-    id "maven-publish"
+    // -----------------------------------------------------------------------------
+    // PUBLISHING
+    // -----------------------------------------------------------------------------
+
+    id 'maven-publish'
+    // Maven Central requires artifacts to be signed.
+    id 'signing'
+}
+
+/*
+ * Maven Central requires artifacts to be released via Sonatype Nexus.
+ * The Nexus Publish Plugin can only be applied to the root project.
+ * Do not apply it if this project is being included as a subproject.
+ */
+def isRootProject = project.path == ':'
+if (isRootProject) {
+    apply plugin: 'io.github.gradle-nexus.publish-plugin'
 }
```

Comment:
You can get rid of this check. It only applies to gradle projects which have subprojects.
So just include the plugin in the `plugins` block and get rid of the buildscript.